src/middleware/assets.js

```diff
@@ -44,10 +43,7 @@ function updater({ serve, base }) {
 export function sync(base) {
   assets.forEach(asset => {
     if (!asset.contents) {
-      asset.contents = readFileSync(base + asset.name);
-    }
-    if (!asset.contentType) {
-      asset.contentType = mime(asset.name);
+      asset.contents = readFileSync(path.join(base, asset.name));
```

Comment:
Should be `asset.file` – in case webpack outputs something that is more than one level deep.